aerospike.logrotate.txt

```diff
@@ -0,0 +1,13 @@
+/var/log/aerospike/aerospike.log {
+    su root root
+    rotate 5
+    dateext
+    size 50M
+    compress
+    olddir /var/log/aerospike/
+    postrotate
+        /bin/kill -HUP `pidof asd`
```

Comment:
please see why asd needs to be restarted? can we logrotate without restarting asd?

Reply:
The above config is directly used from aerospike documentation.

Kill -HUP will not kill the asd process itself, but is required to refresh the handle to log file.